src/nvim/regexp_nfa.c

```diff
@@ -6019,6 +6031,7 @@ static int nfa_regmatch(nfa_regprog_T *prog, nfa_state_T *start, regsubs_T *subm
   log_fd = NULL;
 #endif
 
+theend:
```

Comment:
For reasons unclear to me this label was removed in 13848aadbfed94a62505d4e349426990c75d2ae5, I'm reintroducing it here.